components/meter/meter-mixin.js

```diff
@@ -0,0 +1,50 @@
+import { getLocalizationResource } from '../../tools/localization-load.js';
+import { LocalizeMixin } from '../../mixins/localize-mixin.js';
+
+export const MeterMixin = superclass => class extends LocalizeMixin(superclass) {
+
+	static get properties() {
+		return {
+			max: { type: Number },
+			percent: { type: Boolean },
+			text: { type: String },
+			value: { type: Number }
+		};
+	}
+
+	static async getLocalizeResources(langs) {
+		return getLocalizationResource('meter', langs);
+	}
+
+	constructor() {
+		super();
+		this.max = 100;
+		this.percent = false;
+		this.value = 0;
+	}
+
+	_primary(value, max) {
+		const percentage = max > 0 ? value / max : 0;
+		return this.percent
+			? this.formatNumber(percentage, {style: 'percent', maximumFractionDigits: 0})
+			: this.localize('fraction', 'x', this.value, 'y', this.max);
+	}
+
+	_secondary(value, max, context) {
+		if (!context) {
+			return '';
+		}
+
+		const percentage = this.max > 0 ? value / max : 0;
+		context = context.replace('{%}', this.formatNumber(percentage, {style: 'percent', maximumFractionDigits: 0}));
+		context = context.replace('{x}', value);
+		context = context.replace('{y}', max);
+
+		return context;
+	}
+
+	_ariaLabel(primary, secondary) {
+		const mainLabel = `${primary}, ${this.localize('progressIndicator')}`;
+		return secondary ? `${secondary} , ${mainLabel}` : mainLabel;
```

Comment:
When we're concatenating like this with commas and such, I wonder if those string should also be translated?

Also, is there supposed to be a space after `${secondary}` and before the comma?

Reply:
No space. Thank you.

Good point. I didn't really think about it but we should have a language translation for `{Secondary}, {Primary}, progress indicator`
Or more general `{term1}, {term2}`